python/ray/train/predictor.py

```diff
@@ -76,6 +76,16 @@ class Predictor(abc.ABC):
     def __init__(self, preprocessor: Optional[Preprocessor] = None):
         """Subclasseses must call Predictor.__init__() to set a preprocessor."""
         self._preprocessor: Optional[Preprocessor] = preprocessor
+        # Whether tensor columns should be automatically cast from/to the tensor
+        # extension type at UDF boundaries. This can be overridden by subclasses.
+        self._cast_tensor_columns = False
+
+    @DeveloperAPI
+    def _set_cast_tensor_columns(self):
+        """Enable automatic tensor column casting from/to the tensor extension type at
+        UDF boundaries.
```

Comment:
Nit: First sentence of docstring should be one line

Reply:
Is that aspirational or a strict policy?

Reply:
This is from Google style guide (which is what we follow right?)

> A docstring should be organized as a summary line (one physical line not exceeding 80 characters) terminated by a period, question mark, or exclamation point. When writing more (encouraged), this must be followed by a blank line, followed by the rest of the docstring starting at the same cursor position as the first quote of the first line. There are more formatting guidelines for docstrings below.

Reply:
https://google.github.io/styleguide/pyguide.html

Reply:
Oh even in Black style guide, that adheres to PEP 257: https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
Yeah but I didn't think we were strictly following the one-line summary rule, although maybe we are for AIR abstractions.

Anyways, I made the change!

Reply:
> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. The summary line may be on the same line as the opening quotes or on the next line. The entire docstring is indented the same as the quotes at its first line (see example below).